packages/user/src/components/Invitation/InvitationFormFields.tsx

```diff
@@ -13,19 +13,23 @@ import React, { useEffect, useState } from "react";
 import { InvitationAppOption, InvitationRoleOption } from "@/types";
 
 interface IProperties {
+  additionalFields?: React.ComponentType<{
```

Comment:
This works but IMO it'd be better make this a callback that accept the useFormContext and return the React.ComponentType. A callback will give many options to the dev eg checking some values from useFormContext to return different components.